.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,22 @@
+name: "End-to-end tests"
+
+on:
+  push:
+    branches:
+      - main
+      - releases/*
+  pull_request:
+    branches:
+      - main
+      - releases/*
+
+jobs:
+  e2e-tests:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/setup-go@v2.1.4
+      with:
+        go-version: 1.21
+    - uses: actions/checkout@v2.4.0
+    - name: "Run end-to-end tests"
+      run: make e2e-test INTEGRATION_OUTPUT_JUNIT="true"
```

Comment:
Does `INTEGRATION_OUTPUT_JUNIT` have any effect?

Reply:
Thanks for pointing this out, leftover from copy-paste 😅